src/components/Footer/Footer.module.css

```diff
@@ -0,0 +1,14 @@
+.footer {
+  background-color: #7209b7;
```

Comment:
do you think it's be a good idea to add the colors to variables?

Reply:
also you can group your css properties as it's a little difficult to follow the style - you can choose your own approach or see this suggestion - https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/